primerj/src/main/java/org/primer/primerj/core/TxBuilder.java

```diff
@@ -537,8 +543,8 @@ public Tx buildTx(String changeAddress, List<Out> unspendOuts, Tx tx, Coin... co
         }
```

Comment:
The above block if (needMinFee) {} else {} probably should be removed as primecoin does not have such logic in the new fee protocol. This logic could potentially calculate the fee to be 1 coin when it should be less.